sorc/checkout.sh

```diff
@@ -7,9 +7,10 @@ echo $topdir
 echo fv3gfs checkout ...
 if [[ ! -d fv3gfs.fd ]] ; then
     rm -f ${topdir}/checkout-fv3gfs.log
-    git clone https://github.com/ufs-community/ufs-weather-model fv3gfs.fd >> ${topdir}/checkout-fv3gfs.log 2>&1
+#    git clone https://github.com/ufs-community/ufs-weather-model fv3gfs.fd >> ${topdir}/checkout-fv3gfs.log 2>&1
```

Comment:
I assume the model checkout needs to be updated after development is merged into official GFS v16 location.   If this will be done so, we should update checkout.sh to get the model from the final location.   Jun may not want her fork to become the "goto" location for the GFS v16 model.

Reply:
This will depend on feedback from @junwang-noaa and will be changed shortly either in this PR or directly in gfsv16b depending on when it becomes available.

Reply:
Waiting for new FV3 tag, will update when available.